TestSDK/Smartsheet/Api/Internal/FolderResourcesImplTest.cs

```diff
@@ -40,15 +38,18 @@ public virtual void TestGetFolder()
 
 			//folderResource.getSmartsheet().getHttpClient().close();
 
```

Comment:
This is an invalid test. You are telling the request to include FolderInclude.SOURCE however the JSON file does not include it and there is not a test to see if folder.getSource() is set the to expected value.

Please add source to the json file and then do an assert to validate that the source folder.getSource() is populated with the correct value.